library/src/main/java/in/krharsh17/barview/Bar.java

```diff
@@ -6,23 +6,39 @@
 import android.graphics.drawable.RippleDrawable;
 import android.graphics.drawable.StateListDrawable;
 import android.os.Build;
+import android.util.AttributeSet;
+import android.view.Gravity;
 import android.view.View;
 import android.view.ViewGroup;
 
-class Bar extends View {
-
-    Bar(Context context) {
-        super(context);
-    }
+import androidx.annotation.Nullable;
 
-    void setProgress(float progress) {
+class Bar extends View {
+    void setProgress(float progress,int increaseWidth) {
         ViewGroup.LayoutParams params = this.getLayoutParams();
-        params.width = Math.round(params.width * (progress));
+        params.width   = Math.round(params.width * (progress)) + increaseWidth;
         this.setLayoutParams(params);
         invalidate();
         requestLayout();
     }
+    public Bar(Context context) {
+        super(context);
+//initBackground();
```

Comment:
Format entire code properly

Reply:
unused comments are erased are formatting is completed